.github/workflows/release.yml

```diff
@@ -57,24 +68,32 @@ jobs:
           conda config --set always_yes yes --set auto_update_conda false
           conda install conda=24.1.2
           conda install -n base conda-libmamba-solver
-          conda install python=3.10 conda-build colorama pip ruamel ruamel.yaml rich jsonschema -c conda-forge
-          git fetch --prune --unshallow --tags
-          pip install -e .
+          conda install python=3.12 conda-build colorama pip ruamel ruamel.yaml rich jsonschema uv -c conda-forge
+          uv pip install -e .
 
       # echo yes before login to prevent anaconda bug breaking automation
       # git tags MUST be fetched otherwise output will be blank
       # bash variables cannot be used in github actions, must use actions specific syntax and methods
       # channels need to be specified on build and are saved in the package for installs
       - name: Build the Anaconda Package
         id: condabuild
+        env:
+          NUMPY_VERSION: "1.22.0"
         run: |
           conda install anaconda-client
           conda config --set anaconda_upload no --set solver libmamba
-          echo yes | anaconda login --username ${{ secrets.ANACONDA_CLOUD_USERNAME }} --password ${{ secrets.ANACONDA_CLOUD_PASSWORD }}
-          VERSION_FROM_GIT_TAG=$(git tag --list "v*[0-9]" --sort=version:refname | tail -1 | cut -c 2-) conda build . -c mosek -c anaconda -c pvlib -c slacgismo -c conda-forge --numpy 1.22.0
-          echo "gitversion=$(git tag --list "v*[0-9]" --sort=version:refname | tail -1 | cut -c 2-)" >> $GITHUB_OUTPUT
+          export VERSION_FROM_GIT_TAG=$(git tag --list "v*[0-9]" --sort=version:refname | tail -1 | cut -c 2-)
+          conda build . -c mosek -c anaconda -c pvlib -c slacgismo -c conda-forge --numpy '${NUMPY_VERSION}'
```

Comment:
I think some of this is now outdated (I've added some channels to the latest release)

Reply:
fixed